src/components/Editor/GutterMenu.js

```diff
@@ -139,16 +140,36 @@ class GutterContextMenuComponent extends Component {
     const { contextMenu, ...props } = nextProps;
     const { event } = contextMenu;
     const sourceId = props.selectedSource ? props.selectedSource.id : "";
-    const line = lineAtHeight(props.editor, sourceId, event);
+
+    const classMatch = event.target.className.match(/cl-([0-9]+)-([0-9]+)/);
+    let line = 0;
+    let column = undefined;
+    if (classMatch) {
+      line = classMatch[1];
+      column = classMatch[2];
+    } else {
+      line = lineAtHeight(props.editor, sourceId, event);
+    }
```

Comment:
yikes...

okay - so two things

1. if the user right clicks anywhere in the line we can either not show the breakpoint options or use the first pause point colum
2. if the user right clicks on a column breakpoint we can intercept use the line/column of that column breakpoint

This regex seems evil... and lets find a way not to rely on codemirror classes...